SU-SWS/saa_victoria_subtheme
# ADAPT-000: Added 1em padding to match stanford basic.

# READY FOR REVIEW
# Summary
- ADDING back in the padding left that stanford basic uses

# Review By (Date)
- 7/21

# Urgency
- low

# Steps to Test

1. pull down
1. clear the cache
2. check bullets and ordered lists that they align not stick out to the left.

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
- JIRA ticket
- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
